Sources/Player/PlaybackEngine/PlayerEngine.swift

```diff
@@ -527,11 +527,13 @@ private extension PlayerEngine {
 
 	func handle(_ error: Error, in playerItem: PlayerItem) {
 		guard let notificationsHandler else {
+			PlayerWorld.logger?.log(loggable: PlayerLoggable.handleErrorNoNotificationsHandler)
```

Comment:
For context: We use this code path when there is an error while preloading something (notification handler not yet set)
so the error is going to be logged later.

Reply:
This was not in my original plan, but figured to log the different conditions in this error handling func to see if we are potentially not handling something properly. Your suggestion is to delete this line completely?

Reply:
I was just adding context. I think we can keep it as long as it is not being triggered too many times.

Reply:
luckily we can disable if we want 🤓